Traceback (most recent call last):
  File "/Applications/PyCharm CE (Previous).app/Contents/plugins/python-ce/helpers/pydev/pydevd.py", line 1483, in _exec
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "/Applications/PyCharm CE (Previous).app/Contents/plugins/python-ce/helpers/pydev/_pydev_imps/_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "/Users/clementine/Documents/UCL/NeuralPlayground/neuralplayground/agents/class.py", line 279, in <module>
    agent.run()
  File "/Users/clementine/Documents/UCL/NeuralPlayground/neuralplayground/agents/class.py", line 173, in run
    self.global_test = self.global_test+1
AttributeError: 'Stachenfel2023' object has no attribute 'global_test'. Did you mean: 'global_steps'?
